libraries/H5P.Dialogcards-1.8/dist/h5p-dialogcards.css

```diff
@@ -0,0 +1 @@
cards .h5p-joubelui-button.h5p-dialogcards-disabled{display:none}.h5p-dialogcards .h5p-audio-minimal-button{border:none;background:#1a73d9;font-weight:400;padding:0;font-size:1em;width:1.5em;height:1.5em;line-height:1.5em}.h5p-dialogcards .h5p-audio-minimal-button:before{font-size:.8em;vertical-align:bottom}.h5p-dialogcards .h5p-audio-minimal-button.h5p-audio-minimal-pause:before{font-size:.6em}.h5p-dialogcards .h5p-audio-minimal-button:focus,.h5p-dialogcards .h5p-audio-minimal-button:hover{background:#1356a3}.h5p-course-presentation .h5p-element .h5p-dialogcards-outer-element .h5p-element-inner.h5p-dialogcards.h5p-text-scaling{overflow-y:hidden}.h5p-dialogcards-at-progress,.h5p-dialogcards-card-side-announcer{position:absolute;height:0;width:0;overflow:hidden}.h5p-dialogcards-summary-screen{display:flex;flex-direction:column;align-items:center;padding:1em}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-container{width:100%;display:flex;flex-direction:column;justify-content:center;margin-bottom:2em}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-header{width:100%;font-weight:700;font-size:1.2em;text-align:center}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-subheader{padding-top:.5em;margin-bottom:2em;width:100%;text-align:center}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-message{width:100%;font-size:1.2em;margin-top:1em;text-align:center;font-weight:700;color:#1a73d9}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-footer{width:100%;margin-top:2em;display:flex;flex-direction:row;justify-content:center}.h5p-dialogcards-summary-screen .h5p-dialogcards-button-next-round:before{font-family:H5PFontAwesome4;content:"\F061"}.h5p-dialogcards-summary-screen .h5p-dialogcards-button-restart:before{font-family:H5PFontAwesome4;content:"\F01E"}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table{width:100%}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td{height:1.6em;border-bottom:1px solid #d9d9d9;text-align:left}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-category{width:100%}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol{padding-left:.5em}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol.h5p-dialogcards-check:before{font-family:H5PFontAwesome4;content:"\F00C";color:#1f824c}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol.h5p-dialogcards-times:before{font-family:H5PFontAwesome4;content:"\F00D";color:#d95354}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-score{padding-left:.5em;text-align:right;font-weight:700}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-score-divider{color:#757575;font-size:.75em}.h5p-dialogcards-gone{display:none;visibility:hidden}
```

Comment:
This is packed, but note `.h5p-dialogcards-progress{color:#999` will revert to the non-accessible color.